### PR TITLE
PG: Updates to allow autoconfig based on memory

### PR DESCRIPTION
**Pull request by jamessewell**
new config file created in run hook, then loaded in postgresql.conf through an include

### CODE REVIEW

File: postgresql/hooks/run [hunk 1]
```diff
@@ -2,14 +2,23 @@
 
 set -e
 
+exec 2>&1
+
 echo "Starting PostgreSQL"
 
-export PGDATA={{pkg.svc_data_path}}
+export PGDATA=/hab/svc/postgresql/data
```
Comment (bodymindarts) [inline postgresql/hooks/run]: Please don't hard code this, use the `{{}}` handlebars interpolation. The data dir in perticular may become configurable in a later hab version to support different mount points for persistent volumes.

File: postgresql/hooks/run [hunk 2]
```diff
@@ -2,14 +2,23 @@
...
 
-{{#if svc.me.follower ~}}
-cp {{pkg.svc_config_path}}/recovery.conf {{pkg.svc_data_path}}/
-{{~/if}}
+cat > /hab/svc/postgresql/config/postgresql.local.conf <<-LOCAL
```
Comment (bodymindarts) [inline postgresql/hooks/run]: This should be extracted into a function, but that can be done later.

**Review by bodymindarts — CHANGES_REQUESTED** [review]
Please use the `{{}}` interpolation for the directories.. the rest looks good!

**Comment by jamessewell** [comment]
Updated to new request

**Pull request closed without merging**